src/pyvesync/vesyncoutlet.py

```diff
@@ -105,84 +105,78 @@ def yearly_energy_total(self) -> float:
 
     def display(self):
         super(VeSyncOutlet, self).display()
-        print("\tActive Time: {} minutes, Energy: {}kWh, "
-              "Power: {} watts, Voltage {}".format(
-                  self.active_time, self.energy_today,
-                  self.power, self.voltage))
-        print("\tEnergy - Week: {}kWh, Month: {}kWh, Year: {}kWh".format(
-                  self.weekly_energy_total, self.monthly_energy_total,
-                  self.yearly_energy_total))
+        disp1 = [("Active Time : ", self.active_time, ' minutes'),
+                 ("Energy: ", self.energy_today, " kWh"),
+                 ("Power: ", self.power, " Watts"),
+                 ("Voltage: ", self.voltage, " Volts"),
+                 ("Energy Week: ", self.weekly_energy_total, " kWh"),
+                 ("Energy Month: ", self.monthly_energy_total, " kWh"),
+                 ("Energy Year: ", self.yearly_energy_total, " kWh")]
+        for line in disp1:
+            print("{:.<15} {} {}".format(line[0], line[1], line[2]))
 
 
 class VeSyncOutlet7A(VeSyncOutlet):
     def __init__(self, details, manager):
         super(VeSyncOutlet7A, self).__init__(details, manager)
 
     def get_details(self):
-        r, _ = helpers.call_api(
-            '/v1/device/' + self.cid + '/detail',
-            'get',
-            headers=helpers.req_headers(self.manager)
-        )
+        r, _ = helpers.call_api('/v1/device/' + self.cid + '/detail',
```

Comment:
why all these changes? the previous form is still valid on a lot of these changes and all the changes are making the PR huge

Reply:
I agree, I reverted all of the changes in your following comments and made the call_api formatting consistent throughout the code.  I'll try to keep the PR's smaller in the future